Resources/Prototypes/_SpaceCats/Catalog/Cargo/cargo_service.yml

```
@@ -0,0 +1,9 @@
+- type: cargoProduct
+  id: ServiceBookPrinterCartridge
+  icon:
+    sprite: _SpaceCats/Objects/Misc/refill_cartridge.rsi
+    state: icon
+  product: CrateServicePrinterInkCartridge
+  cost: 1500
+  category: Service
+  group: market
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)